backend/corpora/dataset_processing/process.py

```diff
@@ -46,6 +64,23 @@ def fix_dropbox_url(url):
     return pr.geturl()
 
 
+def update_db(metadata=None, processing_status=None):
+
+    dataset = Dataset.get(os.environ["DATASET_ID"])
+
+    if metadata:
+
+        # TODO: Delete this line one mean_genes_per_cell is in the db
```

Comment:
What does this mean?

Reply:
```suggestion
        # TODO: Delete this line once mean_genes_per_cell is in the db
```

Reply:
Currently there's no column for this in the DB. There will be one, and then we should start populating it.